great_expectations_cloud/agent/agent.py

```diff
@@ -77,8 +80,10 @@ class GXAgent:
 
     def __init__(self: Self):
         agent_version: str = self._get_current_gx_agent_version()
+        print(f"GX Agent version: {agent_version}")
```

Comment:
Should we remove the `agent_version` from line 84 then so it's not printed twice?

Reply:
I agree